### ISSUE TITLE
Proposal: Switch to using OnePassword instead of LastPass

### ISSUE THREAD

**Issue by blag** [StackStorm/discussions#9]
# TL;DR #

LastPass stinks. Let's use 1Password instead.
![Billy Mays sparkling stars](https://i.giphy.com/media/x0d72vcx8mIeUplk4c/giphy.gif)

# Background #

![billy-mays-beating-car](https://user-images.githubusercontent.com/597113/80770470-45c72d80-8b05-11ea-9505-c165a5ba348e.gif)

LastPass is pretty much the worst. It doesn't integrate well, it takes a million clicks to do the most simple task, and it's difficult to manage personal vs. paid-for accounts together. Pretty much everybody hates using LastPass.

# A Fantastic Product #

![Billy Mays here with another fantastic product](https://i.giphy.com/media/l0HlOrRj8sqK1xiJa/giphy.gif)

1Password, on the other hand, is a joy to use. They support Mac, Windows, iOS, Android, Linux, Chrome OS and web, and they have browser extensions for all major browsers. 1Password autofills (and can autosubmit) forms for you, has a unique key combo (`cmd`+`\` or `ctrl`+`\`), you can have it store and autofill more than just a username/password combo (LP, to my knowledge, does not support this) since it supports filling custom fields. They have different vaults, just like LP, that can be shared in the team. It's light, it's fast, and it gets and stays out of your way. They've had their code reviewed, and they take security seriously.

# (If We) Order Right Now! #

![Billy Mays - Order Right Now!](https://i.imgur.com/bcgJf8V.gif)

1Password also supports [open source projects for free](https://github.com/1Password/1password-teams-open-source) with minimal hassle. We can get it in just two easy "payments":

1. submit a PR to repo indicating that we use it (aka free marketing for 1Password)
2. shoot them an email to renew 30 days before the 1-year license ends

We will have to qualify every year for the license extension, and there are some restrictions on the StackStorm organization.

# But Wait, There's More #

![Billy Mays - But Wait, There's More!](https://user-images.githubusercontent.com/597113/80769944-a3f31100-8b03-11ea-87a7-fc967a1abc22.gif)

I think 1Password will also import directly from LastPass exports:
https://support.1password.com/import-lastpass/

This means that we can transition in three easy steps:

1. Export the Shared-Operations vault (and possibly your personal vault) from LastPass.
2. Import the vault/s into 1Password.
3. Party!

# The Only Remaining Question #

At this point the only remaining question is: why haven't we switched to it yet? It's gonna be great!

![Billy Mays - I Guarantee It](https://i.kym-cdn.com/photos/images/newsfeed/001/422/104/93a.jpg)
